Gruntfile.js

```diff
@@ -1003,7 +1003,7 @@ module.exports = function(grunt) {
 						WORKING_DIR + 'wp-{admin,includes}/**/*.js',
 						WORKING_DIR + 'wp-content/themes/twenty*/**/*.js',
 						'!' + WORKING_DIR + 'wp-content/themes/twenty*/node_modules/**/*.js',
-						'!' + WORKING_DIR + 'wp-includes/js/dist/vendor/*.js',
+						'!' + WORKING_DIR + 'wp-includes/js/dist/**/*.js',
```

Comment:
We no longer publish ES5 code to npm. All WordPress packages work with ES2015+ from now on.

Reply:
Can you clarify the change a bit more :p

Reply:
This excludes WP packages from JSHint validation. `js/dist` folder contains files built from `node_modules` with webpack.